iceoryx_hoofs/source/posix_wrapper/signal_handler.cpp

```diff
@@ -57,7 +58,9 @@ void SignalGuard::restorePreviousAction() noexcept
 
 SignalGuard registerSignalHandler(const Signal signal, const SignalHandlerCallback_t callback) noexcept
 {
-    struct sigaction action;
+    struct sigaction action
```

Comment:
Please write `sigaction action{};` to solve formatting issue.

Reply:
Please don't disable formatting if there are other viable options. We are very hesitant to disable clang-format and currently it is only disabled in a few places. When we do it all over the place everybody will do it which will result in a mess.